scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx

```diff
@@ -47,13 +49,17 @@ interface IIMatricsFields {
     };
 }
 
-type IEditableData = {original: IAutoTaggingResponse; changes: IAutoTaggingResponse};
+type IEditableData = {
```

Comment:
Could you revert lines that were modified, but nothing functional was actually changed, like here? Also whitespaces like added/removed newlines in other places.

Reply:
This is now reverted. Sorry about this. We didn't plan to modify it, it was because "ERROR: scripts/extensions/auto-tagging-widget/src/auto-tagging.tsx[53, 36]: trailing whitespace" was shown. Thanks for the feedback!